rollup.config.js

```diff
@@ -1,10 +1,37 @@
 import buble from 'rollup-plugin-buble';
+import uglify from 'rollup-plugin-uglify';
+import zopfli from 'rollup-plugin-zopfli';
 import flow from 'rollup-plugin-flow';
 
-export default {
-	useStrict: false,
-	plugins: [
-    flow(),
-		buble()
-	]
+let config = {
+	entry: 'src/index.js'
 };
+
+if (process.env.BROWSER) {
+	config = Object.assign(config, {
+		dest: 'dist/mitt.umd.js',
+		format: 'umd',
+		moduleName: 'mitt',
+		useStrict: false,
+		sourceMap: true,
+		plugins: [
+			flow(),
+			buble(),
+			uglify({ compress: { warnings: false } }),
+			zopfli({ options: { numiterations: 1000 } })
```

Comment:
Most people use the cjs bundle since that is what webpack uses (either as-is or generated from the ESM/UMD). To me that's the closest approximation of bundle size increase for a lib.